NitroxClient/GameLogic/Entities.cs

```diff
@@ -7,7 +7,6 @@
 using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
 using NitroxModel.DataStructures.Util;
 using NitroxModel.Helper;
-using NitroxModel.Logger;
```

Comment:
This breaks the `Log.Error` inside `DEBUG` down below, at least on Linux:

```
  GameLogic/Entities.cs(144,17): error CS0103: The name 'Log' does not exist in the current context [/home/marijn/Code/C#/Nitrox/NitroxClient/NitroxClient.csproj]
```

Am I doing something stupid or does no-one ever build `DEBUG` anymore?

Reply:
Nope this is a known problem fixed in #1260 Apparently this was due to the ide building in release doesnt see this as a build error unlike within the debug build due to the debug tags the rap the log in this file.

Reply:
Ah great, thanks for showing that!